src/Components/ArtsyShippingEstimate.tsx

```diff
@@ -0,0 +1,348 @@
+import type { EstimateBody } from "@artaio/arta-browser"
+import type {
+  ArtaLocation,
+  ArtaObject,
+  SupportedCurrency,
+} from "@artaio/arta-browser/dist/MetadataTypes"
+import type ArtaEstimate from "@artaio/arta-browser/dist/estimate"
+import { Link, Spacer, Text } from "@artsy/palette"
+import { useLoadScript } from "Utils/Hooks/useLoadScript"
+import { getENV } from "Utils/getENV"
+import type { ArtsyShippingEstimate_artwork$key } from "__generated__/ArtsyShippingEstimate_artwork.graphql"
+import { useEffect, useState } from "react"
+import { graphql, useFragment } from "react-relay"
+import styled from "styled-components"
+
+const ARTA_API_KEY = getENV("ARTA_API_KEY")
+
+const WIDGET_TITLE = "Estimate Shipping Cost"
+
+type Arta = typeof import("@artaio/arta-browser").default
+
+type EstimateWidgetState = {
+  loaded: boolean
+  widget: ArtaEstimate | null
+}
+
+interface Props {
+  artwork: ArtsyShippingEstimate_artwork$key
+}
+
+const Loader = () => <Spacer y={2} />
+
+const LinkButton = styled(Link)`
+  color: ${props => props.theme.colors.black60};
+  &:hover {
+    text-decoration: underline;
+    color: ${props => props.theme.colors.black100};
+  }
+`
```

Comment:
We should move all of this secondary code below the main component; its typically just noise and rarely updates